document_definition/models/document_definition.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+# © 2017 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import api, fields, models
+
+
+class DocumentDefinition(models.Model):
+    _name = 'document.definition'
+
+    term = fields.Char(help='Term to define', required=True)
```

Comment:
I would say just 'Term' or 'Dictionary term' here, otherwise they refer to each other

Reply:
yes agreed